tools/oci-cli-wrapper/src/crane.rs

```diff
@@ -43,4 +48,46 @@ impl ImageTool for CraneCLI {
             serde_json::from_slice(bytes.as_slice()).context(error::ConfigDeserializeSnafu)?;
         Ok(image_view.config)
     }
+
+    async fn push_oci_archive(&self, path: &Path, uri: &str) -> Result<()> {
+        let temp_dir = TempDir::new_in(path.parent().unwrap()).context(error::CraneTempSnafu)?;
+
+        let mut oci_file = File::open(path).context(error::ArchiveReadSnafu)?;
+
+        let mut oci_archive = TarArchive::new(&mut oci_file);
+        oci_archive
+            .unpack(temp_dir.path())
+            .context(error::ArchiveExtractSnafu)?;
```

Comment:
How much time does this step take? It's sort of nice to have just the tar archive as the output file from `rpm2kit`, but if it's a meaningful speed-up, we could have the unpacked contents be the "preferred" form and create the tar archive as a temporary file in the (already slow) Docker path.

Reply:
tested this with just the `tar` command against a recent build of the core kit. unpack took 16 seconds, and recompressing the contents took 1.5 seconds - so could definitely be an optimization worth taking. `docker` would take even longer to push an image but 1.5 seconds is pretty insignificant comparatively